RocketMan/zookeeper
# Ericg/166 play track

Add ability to associate a URL with a track so that the track can be played by clicking on the link from the track review page. Also add a URL entry field in the library editor's track entry table. Changes include:

   - add URL column to tracknames & colltracknames tables
   - add DB upgrade scripts
   - add URL field to track entry table in library editor
   - change track link review page to use new URL as its href and don't make it an anchor is no URL is provided
   - change Search -> Search Albums/Labels (to help distinguish the two modes used in this view)
  - move action buttons out of the track list table
   - remove some of the duplicate code
   - use single quote in HTML 
   - remove noTables option
   - use regex to identify track<DIGIT> element names

Test Cases:
Check the following for collection and non-collection disks and confirm that the review page is correct
   - add new disk and add tracks with and without URLS
   - edit disk and add tracks with and without URLS
   - confirm operation of action buttons, eg Add Track, More Tracks....
